modules/hcp-virt-taints-tolerations.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-virt-taints-tolerations_{context}"]
+= Custom taints and tolerations
+
+For {hcp} on {VirtProductName}, by default, pods for a hosted cluster tolerate the `control-plane` and `cluster` taints. However, you can also use custom taints on nodes so that hosted clusters can tolerate those taints on a per-hosted-cluster basis by setting `HostedCluster.spec.tolerations`.
+
+:FeatureName: Passing tolerations for a hosted cluster
+include::snippets/technology-preview.adoc[]
+
+.Example configuration
+[source,yaml]
+----
+  spec:
+    tolerations:
+    - effect: NoSchedule
+      key: kubernetes.io/custom
+      operator: Exists
+----
+
+You can also set tolerations on the hosted cluster while you create a cluster by using the `--tolerations` hcp CLI argument.
```

Comment:
Should hcp be in backticks if it's the name of the CLI?
```suggestion
You can also set tolerations on the hosted cluster while you create a cluster by using the `--tolerations` `hcp` CLI argument.
```

Reply:
Good question. Yes, we typically format the hcp CLI name in backticks.